Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1117,14 +1136,21 @@ extension BrowserViewController: BrowserToolbarDelegate {
                     }
                 }
 
-                if let popoverPresentationController = activityViewController.popoverPresentationController {
-                    // Using the button for the sourceView here results in this not showing on iPads.
-                    popoverPresentationController.sourceView = toolbar ?? urlBar
-                    popoverPresentationController.sourceRect = button.frame ?? button.frame
-                    popoverPresentationController.permittedArrowDirections = UIPopoverArrowDirection.Up
-                    popoverPresentationController.delegate = self
+                let setupPopover = {
+                    if let popoverPresentationController = activityViewController.popoverPresentationController {
+                        popoverPresentationController.sourceView = button.superview
```

Comment:
I changed this to simplify the code. Using the sourceView superview always works.